package.json

```diff
@@ -17,16 +26,13 @@
     "build": "yarn build:app",
     "start": "NODE_PATH=./src react-scripts start"
   },
-  "main": "lib/index.js",
-  "files": [
-    "package.json",
-    "README.md",
-    "lib"
-  ],
+  "eslintConfig": {
+    "extends": "react-app"
+  },
   "babel": {
     "presets": [
       "babel-preset-react",
-      "latest"
+      "env"
```

Comment:
`babel-preset-latest` was deprecated in favor of `babel-preset-env` which compiles latest syntax according to the environment you are running in, so you only get compiled what you need (smaller builds).